app/Models/EventLog.php

```diff
@@ -22,14 +21,14 @@ class EventLog extends Model
      * @var array
      */
     protected $fillable = [
-      'cid', 'tid',
-      'publisher_event_id', 'banner_id',
-      'event_type',
-      'pay_to', 'ip',
-      'our_context', 'their_context',
-      'user_id', 'human_score', 'our_userdata', 'their_userdata',
-      'timestamp',
-      'event_value', 'paid_amount', 'payment_id'
+        'cid', 'tid',
+        'publisher_event_id', 'banner_id',
+        'event_type',
+        'pay_to', 'ip',
+        'our_context', 'their_context',
+        'user_id', 'human_score', 'our_userdata', 'their_userdata',
+        'timestamp',
+        'event_value', 'paid_amount', 'payment_id',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "event_value" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)